libagbsyscall/libagbsyscall.s

```diff
@@ -145,22 +145,22 @@ HuffUnComp:
 	.endif
 
 	.ifdef L_SoftResetExram
-	arm_func_start SoftResetExram
+	thumb_func_start SoftResetExram
 SoftResetExram:
 	ldr r3, =REG_IME
 	movs r2, #0
 	strb r2, [r3, #0]
 	ldr r3, =SOFT_RESET_DIRECT_BUF
 	movs r2, #1
 	strb r2, [r3, #0]
-	subs r3, #SOFT_RESET_DIRECT_BUF - 0x3007f00
+	sub r3, #SOFT_RESET_DIRECT_BUF - 0x3007f00
```

Comment:
Why remove the condition suffices? (+#L159)

Reply:
Because this is syntax divided; thumb doesn't use that suffix

Reply:
Then why leave the others (`movs`)?

Reply:
I fixed that too!